.github/workflows/examples-nrfconnect.yaml

```diff
@@ -131,7 +131,7 @@ jobs:
                     /tmp/bloat_reports/
             - name: Build example nRF Connect SDK All Clusters App on nRF52840 DK
               run: |
-                  scripts/examples/nrfconnect_example.sh all-clusters-app nrf52840dk_nrf52840 -DCONF_FILE=prj_dfu.conf
+                  scripts/examples/nrfconnect_example.sh all-clusters-app nrf52840dk_nrf52840
```

Comment:
This does not seem related to this PR....

Reply:
All-clusters-app build for nrf connect was failing due to this ( it was running out of flash ). So Damian suggested to remove this to fix the CI.

Reply:
Yes, but it should have been a separate PR, and that is in fact what landed earlier today....

Reply:
Ok I'll add it in separate PR

Reply:
Ohh its alright merged. I'll rebase on master.